udf-compiler/src/main/scala/com/nvidia/spark/udf/LambdaReflection.scala

```diff
@@ -152,7 +151,17 @@ object LambdaReflection {
   }
 
   def getClass(name: String): Class[_] = {
-    ShimLoader.loadClass(name)
+    // Don't use ShimLoader.loadClass because in the REPL use case the classes
+    // compiled by REPL are accessible via the thread's context classloader,
+    // but not by the parent classloader detected by the plugin.
+    //
+    // This code is executed after the Plugin and ShimLoader has already been initialized.
+    // REPL classes are able to interact with the Plugin classes because the the REPL
```

Comment:
nit: sorry I duplicated the
```suggestion
    // REPL classes are able to interact with the Plugin classes because the REPL
```

Reply:
oh I missed that